test/with_dummyserver/test_socketlevel.py

```diff
@@ -458,7 +458,7 @@ def socket_handler(listener):
 
         self._start_server(socket_handler)
         with HTTPConnectionPool(
-            self.host, self.port, timeout=SHORT_TIMEOUT, retries=True
+            self.host, self.port, timeout=LONG_TIMEOUT, retries=True
```

Comment:
Can we rename this test to match new retry option terminology?

Reply:
Done, thanks